docs/docs/databases/athena.mdx

```diff
@@ -33,10 +33,7 @@ following connection string:
 awsathena+rest://{aws_access_key_id}:{aws_secret_access_key}@athena.{region_name}.amazonaws.com/{schema_name}?s3_staging_dir={s3_staging_dir}&...
```

Comment:
It won't let me mark it on line 29 but that should be "you can also use **the** PyAthena library" -- maybe that can be fixed in this PR even though it's not an addition